### PR TITLE
Email style mods

### PR DESCRIPTION
**Pull request by ericronne**

### CODE REVIEW

**Comment by ericronne** [comment]
@hbillings i've dropped in an svg logo here. Does that present compatibility / implementation issues?

File: data_capture/templates/email/base.html [hunk 1]
```diff
@@ -298,8 +298,8 @@
                   <table border="0" cellpadding="0" cellspacing="0">
                     <tr>
                       <td>
-                        <div class="calc-logo-wrapper align-center">
-                          <img src="{% absolute_static 'frontend/images/calc_logo_blue.png' %}" alt="" width="180" height="73">
+                        <div class="calc-logo-wrapper align-left">
+                          <img src="{% absolute_static 'frontend/images/calc_logo_black.svg' %}" alt="" width="180">
```
Comment (jseppi) [inline data_capture/templates/email/base.html]: I browsers (especially IE) can have problems with SVG when `width` and `height` are not both specified

> If you use an  to embed your SVG, setting height and width will make the SVG scale predictably in most browsers, but not in Internet Explorer. With CSS like img { width: 100%; height: auto; }, IE will auto-scale the image area to keep the width:height aspect ratio constant, but it won't scale the actual drawing to match the scale of the image dimensions.

ref: https://css-tricks.com/scale-svg/

Comment (ericronne) [reply data_capture/templates/email/base.html]: Cool thx

**Comment by ericronne** [comment]
@hbillings (et al) do we have access to Litmus, or another email emulator for testing?

**Comment by ericronne** [comment]
that tool is MWAH

**Comment by toolness** [comment]
What does MWAH mean?  Is it [the paticular sound that a fretless bass guitar makes when played](https://www.urbandictionary.com/define.php?term=mwah)?

**Comment by ericronne** [comment]
whoa who knew?
i am MWAH over the sound of a fretless bass as well

### "MWAH!"
giphy

**Comment by ericronne** [comment]
svg is misbehaving 😠 will debug

image

File: frontend/source/sass/email.scss [hunk 2]
```diff
@@ -39,54 +39,58 @@ a {
...
 
 table.body .btn table {
+  margin: auto;
+  max-width: 300px;
   width: 100%;
 }
 
 table.body .btn a {
+  margin: auto;
+  max-width: 300px;
```
Comment (hbillings) [inline frontend/source/sass/email.scss]: Is this going to be wide enough to not look funky on mobile?

Comment (ericronne) [reply frontend/source/sass/email.scss]: i _think_ so??
this is @400 wide, but it's from desktop; don't know how to actually preview on mobile

image

Comment (hbillings) [reply frontend/source/sass/email.scss]: I _think_ you might be able to preview this on iOS simulator: https://gist.github.com/shawnbot/5527355

Comment (ericronne) [reply frontend/source/sass/email.scss]: unauthorized app :(

Comment (ericronne) [reply frontend/source/sass/email.scss]: ooh installed xcode!
but …
image

maybe we should find a pc somewhere

Comment (hbillings) [reply frontend/source/sass/email.scss]: You're viewing the source; what does the `HTML` tab show?

Comment (ericronne) [reply frontend/source/sass/email.scss]: hrm that was the source view, here's with the styles.
we really need to test with an email client.

image

**Comment by ericronne** [comment]
Ok simulator is cool, but not a great way for us to QA these emails.

Per @jseppi:
Authorized users can upload a dummy price list to [calc-dev.app.cloud.gov](http://calc-dev.app.cloud.gov/), and then reject it in the Admin section. This will generate an email to the user.

**Comment by ericronne** [comment]
Closed #1782

**Pull request merged**